docs/comparison-of-builders.md

```diff
@@ -0,0 +1,221 @@
+# Comparison of DiCy to other LaTeX Builders
+
+There are many different automatic and semi-automatic builders for LaTeX-centric
+documents. The [Compilation Topic][] at CTAN provides a current list of some of
+those specifically designed for TeX or LaTeX documents. In addition there are
```

Comment:
Pandoc internally uses LaTeX, but I don't understand the intention of mentioning pandoc here.